core/prob_dist.py

```diff
@@ -32,6 +32,24 @@ def alphabet(self):
     def prob_list(self):
         return [self.prob_dict[s] for s in self.alphabet]
 
+    @classmethod
+    def get_sorted_prob_dist(cls, prob_dict, reverse=True):
```

Comment:
I don't like making `reverse=True` default - we should follow the standard interface to sort in Python and have it as False by default

Reply:
Sorry I don't understand this? Why would you do that if we want in reverse sorted order and we are using a built-in function?

Reply:
@tpulkit: I kind of agree with shubham. Also, the function name`get_sorted_prob_dist` intuitively implies sorting in increasing order.. so maybe we should keep that as the default.

Reply:
make default parameter False, and then call appropriately